src/main/resources/application-development.properties

```diff
@@ -5,8 +5,10 @@ spring.datasource.username=sa
 spring.datasource.password=password
 spring.h2.console.settings.web-allow-others=true
 spring.h2.console.enabled=true
-app.showSwaggerUILink=true
+# app.showSwaggerUILink=true
```

Comment:
Please remove the commented out line of code.

And team members: please do not put LGTM on a PR that has commented out lines in it.